Cell.xctemplate/___FILEBASENAME___CellModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ___FILENAME___
+//  ___PROJECTNAME___
+//
+//  Created by ___FULLUSERNAME___ on ___DATE___.
+//  Copyright © ___YEAR___ ___ORGANIZATIONNAME___. All rights reserved.
+//
+
+import UIKit
+
+class ___FILEBASENAMEASIDENTIFIER___CellModel {
+
+    init() {
```

Comment:
Jaký přesně má ten `init` význam?

Reply:
v 90% pripadoch ma CellModel init() do kt davame rozne properties, ale hej asi je to zbytocne